### PR TITLE
ThriftyNova updated to 2025

### PR DESCRIPTION
**Pull request by clrozeboom**

### CODE REVIEW

**Comment by thenetworkgrinch** [comment]
There have been alot of changes that this PR re-applies. Can you copy your ThriftyNova file over and make the necessary changes to `DeviceJson` then make a separate PR for your swerve drive test changes?

**Comment by clrozeboom** [comment]
Whoops, yeah I've been trying to get this finalized so that it's out of my way. Is this better?

File: src/main/java/swervelib/parser/json/DeviceJson.java [hunk 1]
```diff
@@ -221,6 +226,8 @@ public SwerveMotor createMotor(boolean isDriveMotor)
             // We are creating a motor for an angle motor which will use the absolute encoder attached to the data port.
             return new SparkMaxBrushedMotorSwerve(id, isDriveMotor, Type.kNoSensor, 0, false, DCMotor.getCIM(1));
         }
+      case "thrifty_nova":
```
Comment (thenetworkgrinch) [inline src/main/java/swervelib/parser/json/DeviceJson.java:229]: Needs one per motor type that can be attached to the Thrifty Nova. Should create the DCMotor here too.

**Review by thenetworkgrinch — CHANGES_REQUESTED** [review]
Looks great! Just a little details that are needed. Thank you!

**Pull request merged**